corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less

```diff
@@ -1,24 +1,19 @@
-.flex-container {
-  display: flex;
-  align-items: stretch;
-}
-
 .full-width {
   width: 100%;
 }
 
-.flex-child {
-  flex: 1;
-  min-width: 200px;
-  &:empty {
-    display: none;
-  }
+.sidebar-push {
+  margin-left: 310px;
 }
 
-.flex-child-sidebar {
-  max-width: 400px;
+#sidebar-region {
+  background: transparent;
+  width: 300px;
+  position: absolute;
```

Comment:
I personally think this is nicer than a custom flexbox layout, and gives a predictably sized sidebar which is a pretty common UI choice. That said, if we truly believe the sidebar needs to grow and shrink on medium-to-large sized screens I would opt for a bootstrap implementation, something like applying `col-md-3` and `col-md-9` to the sidebar and case list regions, respectively.